README.md

```diff
@@ -90,12 +90,23 @@ Running Tests
 
 There are three major test-suites you may be interested in running.
 
-The unit tests (running individual `*.tzt` programs and checking their output):
+The unit tests (running individual `*.tzt` programs and checking their exit code):
 
 ```sh
 make test-unit -j8
 ```
 
+The symbolic unit tests (running individual `*.tzt` programs and checking their
+output using `lib/michelson-test-check`). Note that their are three flavours of
+symbolic unit tests. Those with the `stuck.tzt` extension, test ill-formed
+programs and are expected to get stuck before completing exectution. Those with
+the `fail.tzt` extension, are "broken" tests where assertions are expected to
+fail.
```

Comment:
@nishantjr are they broken as in they currently don't work but eventually should work? Then please use a failing test list at `tests/failing.symbolic` to exclude them instead.

Reply:
Having the clear list of failing tests instead of relying on `fail.tzt` makes it a lot easier to see progress on making the tests work (esp when reviewing git deltas).

Reply:
No, these are should always fail. I think a better way of talking about it is that the tests passes when the the definition returns a non-zero exit code. Since we are writing a test framework, we have tests where assertions fail to make sure our assertion code actually does what we think it does.

Reply:
Ah makes sense.